AutomatedTesting/Gem/PythonTests/AWS/common/constants.py

```diff
@@ -0,0 +1,13 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+ASSUME_ROLE_ARN = 'arn:aws:iam::645075835648:role/o3de-automation-tests'

```

Comment:
Minor: These should have comments as to what they set

Reply:
Sure. Will add description for the constants.